tools/bbbconfgen/README.md

```diff
@@ -0,0 +1,101 @@
+# bbbconfgen
+
+Application to generate text files from a template, replacing placeholders that specify Redis keys.
+It's written in Go as part of the [BitBox Base](https://github.com/digitalbitbox/bitbox-base) project by [Shift Cryptosecurity](https://shiftcrypto.ch) and used for automatically generating configuration files.
+
+The program reads a text file specified using the `--template` argument, parses the contents and writes it into a target file that is either specified with the `--output` argument, or directly on the first line of the template.
+
+## Usage
+
+The program is run from the command line.
+
+```console
+$ bbbconfgen --help
+
+bbbconfgen version 0.1
+generates text files from a template, substituting placeholders with Redis values
+
+Command-line arguments:
+  --template      input template text file
+  --output        output text file
+  --redis-addr    redis connection address (default "localhost:6379")
+  --redis-db      redis database number
+  --redis-pass    redis password
+  --verbose
```

Comment:
See comment above on `--verbose` and `--quite`.

Reply:
Done.